package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "publish-to-apple-news",
-  "version": "2.4.0",
+  "version": "2.4.1",
```

Comment:
Bumping the plugin version here will make the plugin version out of step with the plugin's headers (which WordPress uses to manage the version). To mirror `create-wordpress-plugin`, we should also update the plugin's headers here:

- https://github.com/alleyinteractive/apple-news/blob/develop/apple-news.php#L17
- https://github.com/alleyinteractive/apple-news/blob/develop/readme.txt#L8

That is what the `npm run release` command should do, too.

Reply:
on it